plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -194,7 +198,7 @@ func shouldIgnore(a admission.Attributes) (bool, error) {
 // computeSecurityContext derives a valid security context while trying to avoid any changes to the given pod. I.e.
 // if there is a matching policy with the same security context as given, it will be reused. If there is no
 // matching policy the returned pod will be nil and the pspName empty.
-func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes, pod *api.Pod, specMutationAllowed bool) (*api.Pod, string, field.ErrorList, error) {
+func (c *PodSecurityPolicyPlugin) computeSecurityContext(a admission.Attributes, pod *api.Pod, specMutationAllowed bool, ValidatedPSP string) (*api.Pod, string, field.ErrorList, error) {
```

Comment:
Why argument in upper case?
s/ValidatedPSP/validatedPSP/

Reply:
Done.